.github/workflows/CI.yml

```diff
@@ -40,11 +40,8 @@ jobs:
       run: ./build ci
       if: ${{ runner.os != 'Windows' }}
 
-    - name: Test - Persistence
-      run: dotnet bin/Debug/Persistence.Tests/Persistence.Tests.dll
-
-    - name: Test - PAModel
-      run: dotnet bin/Debug/PAModelTests/PAModelTests.dll
```

Comment:
@joem-msft FYI the assembly YamlValidator.Tests.dll wasn't run in CI. It's better to `dotnet test` on the solution rather than listing individual dlls.

Reply:
please do not start including the YamlValidator.Tests in the CI builds, as this project has been abandoned and not maintained.

Reply:
Done.